packages/builder/src/pages/builder/portal/manage/auth/index.svelte

```diff
@@ -392,15 +408,135 @@
       <div class="form-row">
         <Label size="L">Activated</Label>
         <Toggle
+          dataCy={"oidc-active"}
           text=""
           bind:value={providers.oidc.config.configs[0].activated}
         />
       </div>
     </Layout>
+    <span class="advanced-config">
+      <Layout gap="XS" noPadding>
+        <Heading size="XS">
+          <div class="auth-scopes">
+            <div>Advanced</div>
+            <Button
+              secondary
+              newStyles
+              size="S"
+              on:click={() => {
+                providers.oidc.config.configs[0]["scopes"] = [...defaultScopes]
```

Comment:
Why do we always need to use the `0` index here? Just wondering if there's a potential for a breakage here since it's just accessing the first element regardless

Reply:
We have potential support for having multiple OIDC configs at once, but the screen itself seems to be intended to allow only one at the minute. My assumption was that work to display multiple configs was parked as some point.

Reply:
That's right, the original backend was designed to be able to add multiple oidc configs if desired, mostly for backwards compatibility on the document schema if someone ever did want to add multiple auth sources. The UI is coded to work with one, but it would be possible to have multiple in future with some refactoring.